examples/tuple-match-statement/tuple_match_statement.bal

```diff
@@ -28,40 +28,38 @@ public function main() {
     matchWithTypeGuard(b5);
 }
 
-// This method uses structured tuple match patterns with different sizes. The given `match` expression
-// will be checked for "isLike" relationship and will be matched at runtime.
+// This method uses tuple match patterns of different sizes. The `match` expression `a`
+// will be matched against the given pattern list at run time based on the
+// "is-like" relationship between the expression and a pattern.
 function basicMatch(any a) {
     match a {
-        // This pattern check is for a tuple type of three variables and types can be of any.
-        var (s, i, b) => io:println("Matched with three vars : "
-                                    + io:sprintf("%s", a));
-        // This pattern check is for a tuple type of two variables and types can be of any.
-        var (s, i) => io:println("Matched with two vars : "
-                                    + io:sprintf("%s", a));
-        // This pattern check is for a single variable which can be of type `any`. This has to be the last pattern.
-        var s => io:println("Matched with single var : "
-                                    + io:sprintf("%s", a));
+        // This pattern check is for a tuple of three members of any type.
+        var [s, i, b] => io:println("Matched with three vars : ",  io:sprintf("%s", a));
+        // This pattern check is for a tuple of two members of any type.
+        var [s, i] => io:println("Matched with two vars : ",  io:sprintf("%s", a));
+        // This pattern check is for a single variable, which can be of type `any`. This has to be the last pattern.
+        var s => io:println("Matched with single var : ", io:sprintf("%s", a));
     }
 }
 
-// This method uses structured tuple match patterns with different sizes along with type guards. The given
-// `match` expression will be checked for "isLike" relationship and also it will check the type guard for the pattern
+// This method uses structured tuple match patterns with different sizes along with match guards. The given
+// `match` expression will be checked for "isLike" relationship and also it will check the match guard for the pattern
```

Comment:
```suggestion
// `match` expression will be checked for "is-like" relationship and also it will check the match guard for the pattern
```

since L33 says so